styles.css

```diff
@@ -0,0 +1,57 @@
+#welcome-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    padding: 50px;
+    height: 100vh;
+    overflow: auto;
+    background-color: burlywood;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+}
+.navbar{
+    display: flex;
+    align-items: center;
+    justify-content: right;
+    position: fixed;
+
+}
```

Comment:
Please remove extra lines (as on line 21) to make your code clean. And also leave a new line before line 16.

Reply:
HI @tahseenzahra , Good suggestions 👌, but Remember to include emojis to make your code review more human-friendly 👽

Reply:
@Bondok6 Hello ya I wanted to but I can't see the icon to add emoji and I have seen that it is being used manually and I do struggle with that.

Reply:
@tahseenzahra, ah It's ok 👌
> I'll tell you a tip if you're using Windows 
> Type Windows logo key `Win` + . (period). The emoji keyboard will appear. 👌 [Check This](https://support.microsoft.com/en-us/windows/windows-keyboard-tips-and-tricks-588e0b72-0fff-6d3f-aeee-6e5116097942)